enhancements/olm/platform-operators.md

```diff
@@ -0,0 +1,609 @@
+- Detach optional, CVO-based operators from the [payload lifecycle](#payload-lifecycle) during phase 0.
+- Enable all OLM-based operators to be deployed via the PO mechanism during phase 0.
+- Support any essential operator to be deployed via the PO mechanism. Future phase designs may re-evaluate this agent-based approach.
+- Support for extending the PO mechanism to cover any MAPI/CCM use cases that will enable supplementing OpenShift with platform-specific integrations during phase 0.
+- Support sourcing catalog content from any CatalogSource in the cluster during phase 0.
+- Support dependency resolution for PO packages during phase 0.
+- Support automatic upgrades for PO packages during phase 0.
+- Support more fine-tuned health checks during phase 0.
+- Require PO packages to report state through an auxiliary API (e.g. ClusterOperator, OperatorCondition, etc.) during phase 0.
+- Provide a mechanism for admins to manually approve PO package upgrades during phase 0.
+- Permit non-essential operators to be rolled back.
+
+## Terminology
+
+### Payload Lifecycle
+
+These are components that build/ship with the OCP payload as a unit. This unit includes both core, control plane components, in addition to the independent, "value" add components that all release on the same release cadence.
+
+### Cluster Lifecycle
+
+These are components that are automatically upgraded when the cluster gets upgraded, and can block cluster lifecycle events:
+
+- Block cluster rollout when an individual component cannot successfully install
+- Block cluster upgrades when an individual component cannot tolerate the incoming version, or has another reason to block the upgrade.
+
+These are the components that install and upgrade with the OCP payload, and influence cluster lifecycle events noted above.
+
+### Non-Essential Operators
+
+These are payload components that don't contribute to a working OpenShift control plane.
+
+### Plain Bundle Format
+
+A "plain" bundle format is OLM's newest bundle format that supports packaging arbitrary Kubernetes resources. This bundle format can also be referred to as the plain+v0 bundle format.
+
+### Resolveset Bundle Format
+
+A "resolveset" is an OLM bundle format that expresses the set of bundles that need to exist on the cluster in order to satisfy consumer constraints. While this resolveset bundle format is still in the work-in-progress stage, we can currently define this format as a YAML stream of individual BundleDeployment manifests.
+
+### Bundle/BundleDeployment API(s)
+
+A "BundleDeployment" is an OLM API that's responsible for managing OLM's first-class Bundle API, which represents content that needs to be made available to other consumers on the cluster. Both of these APIs are cluster-scoped resources.
+These APIs are analogous to the Deployment/PodSpec resources seen throughout core Kubernetes. The BundleDeployment API allows users to specify a Bundle template, and that template contains the desired specification
```

Comment:
Nit: Bundle is not analogous to Pod. It's more analogous to `pod.spec.containers[].image`.